### DISCUSSION TITLE
igars777

### DISCUSSION THREAD

**Discussion by igars777** [Chia-Network/chia-blockchain#7772] [Support]
Управление целевыми адресами для получения вознаграждений за фарминг now  working

**Comment by igars777** [comment]
Управление целевыми адресами для получения вознаграждений за фарминг

**Comment by igor19801** [comment]
установил версию  Блокчейн Chia 1.2.7 участки загрузились на 100%, фарминг не загружается, что делать?

**Comment by loppefaaret** [comment]
@igars777 im not entirely sure my translation works correctly - are you saying that setting your "farmer/pool rewards target address" does not work ? 
if you are using plotNFT to farm plots, the pool payouts would have to be "Claim Rewards" under the plotNFT - only OG plots would have use of the "pool rewards target address" under the "Farm" tab

**Comment by BrandtH22** [comment]
@igars777 , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
